### ISSUE TITLE
Sanitization of top domains list

### ISSUE THREAD

**Issue by bbinto** [mozilla-mobile/focus-android#773]
Please update / remove the following domains from the existing top domain lists (per country)

diply.com
pch.com
swagbucks.com
outbrain.com
ijreview.com (down)
ebates.com
swagbucks.com
faithtap.com
littlethings.com
nametests.com
iflscience.com
buzzlie.com (down)
emgn.com
myway.com (search hijack)
ozztube.com (down)
topix.com
cafemom.com
fatwallet.com
shareasale.com (blocked by Disconnect Malvertising list)
tickld.com
zergnet.com
regnok.com (malware)
digitaltrends.com
worldlifestyle.com
wow.com

**Comment by pocmo** [comment]
Is this for v1?

**Comment by bbinto** [comment]
As per our code freeze and engineering email, since this is not a blocker or a crash, it's not v1.

**Comment by liuche** [comment]
Telemetry shows that autocomplete isn't highly used, and we need to go over this list again, per @bbinto.

**Comment by pocmo** [comment]
To my knowledge there's no autocomplete telemetry?

**Comment by bbinto** [comment]
I was going by this

"(Search) Query/Hint clicked: - ("action", "select_query", "search_bar")"
https://github.com/mozilla-mobile/focus-android/blob/master/docs/telemetry.md

<img width="260" alt="screen shot 2017-06-26 at 7 30 59 am" src="https://user-images.githubusercontent.com/122897/27537346-72cc0870-5a41-11e7-9cdb-948216cbe3d4.png">

**Comment by pocmo** [comment]
Oh, I see. This is counting a click on the UI piece that appears below the URL bar:

![searchbar](https://user-images.githubusercontent.com/89638/27538700-8d2c5836-5a79-11e7-93c0-edc2c9b1f344.png)

**Comment by bbinto** [comment]
Ok....so is there a way to track the event I mentioned, i.e. going to the URL after autocomplete was used?

**Comment by pocmo** [comment]
No currently we do not send an event for that. It's a bit tricky to do correctly but not impossible.

**Comment by bbinto** [comment]
Ok, so let's park this then for now.

**Comment by bbinto** [comment]
closing this in favour of #1337
